extension/src/experiments/columns/collect/index.test.ts

```diff
@@ -557,140 +459,153 @@ describe('collectChanges', () => {
   })
 
   it('should not fail when the workspace does not have metrics but a previous commit does', () => {
-    const data: ExperimentsOutput = {
-      [EXPERIMENT_WORKSPACE_ID]: {
-        baseline: {
-          data: {
-            params: mockedExperimentData.baseline.data.params
-          }
-        }
+    const data: ExpShowOutput = generateTestExpShowOutput(
+      {
+        params: mockedMetricsAndParams.params
       },
-      f8a6ee1997b193ebc774837a284081ff9e8dc2d5: mockedExperimentData
-    }
+      {
+        data: mockedMetricsAndParams,
+        rev: 'f8a6ee1997b193ebc774837a284081ff9e8dc2d5'
+      }
+    )
 
     expect(collectChanges(data)).toStrictEqual([])
   })
 
-  const updateParams = (data: ValueTree) => ({
-    baseline: {
-      data: {
-        timestamp: null,
-        params: {
-          'params.yaml': {
-            data
-          }
-        },
-        status: ExperimentStatus.SUCCESS,
-        executor: null
+  const getParams = (data: ValueTree) => ({
+    params: {
+      'params.yaml': {
+        data
       }
     }
   })
 
   it('should work for objects', () => {
     expect(
-      collectChanges({
-        workspace: updateParams({
-          a: { b: 1, d: { e: 100 } }
-        }),
-        '31c419826c6961bc0ec1d3900ac18bf904dcf82e': updateParams({
-          a: { b: 'c', d: { e: 'f' } }
-        })
-      })
+      collectChanges(
+        generateTestExpShowOutput(
+          getParams({
+            a: { b: 1, d: { e: 100 } }
+          }),
+          {
+            data: getParams({
+              a: { b: 'c', d: { e: 'f' } }
+            }),
+            rev: '31c419826c6961bc0ec1d3900ac18bf904dcf82e'
+          }
+        )
+      )
```

Comment:
I'm not a massive fan of how much "magic" is in the setup of this test but WDYT? Should we do this more/not at all?

Reply:
If this starts failing debugging would not be fun

Reply:
While it would be harder to debug, it does help reduce code repetition. I'm ok with doing it more! Might just be me, but would it be more readable if it was broken up more? Took me a sec to figure out what arguments were going where :)